### DISCUSSION TITLE
Can't create new project

### DISCUSSION THREAD

**Discussion by njames06** [WolvenKit/WolvenKit#916] [Q&A] [answered]
I'm trying to create a new project and nothing happens after I enter name, file path and then hit "Create". Literally nothing happens. Have hit the button and let it sit for a few minutes and have spammed the button. Quite literally nothing happens. I have even tried filling out the entire new project form with no change in outcome. Using the newest version released today, 8.7.0-nightly.2022-09-26. Just want to make sure I'm not doing something wrong before I create a bug report. Anyone else experiencing this?

**Comment by glitchered** [comment]
had the same issue. 22-09-24 did not create the project. after clicking the second time it dissappered but stayed in the task manager. updated to 22-09-26 it did the same but atleast created the project. i had to go back to 22-09-24 tho, to actually open and use it. lil weird bug.

**Comment by rfuzzo** [comment, marked as answer]
Thank you very much for your bug reports: this is fixed in https://github.com/WolvenKit/WolvenKit/commit/2a5adccbf07365c3376586b9f082403df76ffef2

**Reply by njames06** [reply]
Thanks rfuzzo! Downloaded the newest release and was able to create a new project.
